target/bin/sedfile

```diff
@@ -26,7 +26,6 @@ then
   exit 1
 fi >&2
 
-[[ -f /CONTAINER_START ]] && SKIP_ERROR=1 # hide error if container was restarted
```

Comment:
You should technically be able to keep this and the two related sedfile test cases? The tests only simulate a restart by adjusting the presence of `CONTAINER_START`. Your fail early check wouldn't conflict with that.

Reply:
Right; I will test this in the last PR of the series after #3115.